### ISSUE TITLE
Missleading comment in config sample file in OC-8

### ISSUE THREAD

**Issue by arkascha** [owncloud/core#14114]
In the cinfog sample file bundled with OC-8 there is this comment: 

```php
/**
 * Check 3rd party apps to make sure they are using the private API and not the
 * public API. If the app uses the private API it cannot be installed.
 */
```

This sounds funny to me, somehow. No wonder there are so few 3rdparty apps :-)

**Comment by PVince81** [comment]
Ah! That explains it!

**Comment by MorrisJobke** [comment]
Fix is in #14586
